js/components/interface/3dCanvas/ThreeDEngine.js

```diff
@@ -28,11 +28,11 @@ define(['jquery'], function () {
   THREE.FilmPass = require('imports-loader?THREE=three!exports-loader?THREE.FilmPass!three\/examples\/js\/postprocessing\/FilmPass');
```

Comment:
@ddelpiano why is this file changed here in this branch? I see mostly formatting but also some actual changes and I thought this is stuff that was a untouched since we solved those other issues months ago

Reply:
there is a single line changed, the canvas was detecting the click when the querybuilder is open but the coordinates were undefined since the action was not performed on the canvas itself, I had to add that check to avoid this issue.

Reply:
Ah ok got it, I forgot and I thought unrelated to query stuff. In particular if there is a singel change trying to not do a lot of formatting at the same time otherwise it's very hard to spot what actually changed 👍